docs/modules/azure.md

```diff
@@ -0,0 +1,65 @@
+# Azure Module
+
+!!! note
+This module is INCUBATING. While it is ready for use and operational in the current version of Testcontainers, it is possible that it may receive breaking changes in the future. See [our contributing guidelines](/contributing/#incubating-modules) for more information on our incubating modules policy.
+
+Testcontainers module for the Microsoft Azure's [SDK](https://github.com/Azure/azure-sdk-for-java).
+
+Currently, the module supports `CosmosDB` emulator. In order to use it, you should use the following class:
+
+Class | Container Image
+-|-
+CosmosDBEmulatorContainer | [mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator](https://github.com/microsoft/containerregistry)
+
+## Usage example
+
+### CosmosDB
+
+Start Azure CosmosDB Emulator during a test:
+
+<!--codeinclude-->
+[Starting a Azure CosmosDB Emulator container](../../modules/azure/src/test/java/org/testcontainers/containers/CosmosDBEmulatorContainerTest.java) inside_block:emulatorContainer
+<!--/codeinclude-->
+
+Prepare KeyStore to use for SSL. 
+**Keep in mind that, certificate path can be different across docker image versions** 
```

Comment:
I don't think this one is relevant anymore :)

Reply:
yep 👍